release notes/v0.26.0.md

```diff
@@ -0,0 +1,121 @@
+
+Thanks, @KajdeMunter!
+
+Also the k6 Dockerfile alpine version was bumped to 3.10. Thanks @dmitrytokarev!
+
+## UX
+
+* Better timeout messages for `setup` and `teardown` timeouts including a hint on how to fix it. (#1173)
+* When a folder is passed with `open()`, the resulting error message will now include the path to the specified folder. (#1238)
+* K6 Version will now include the git commit it was build from (in most cases ...). (#1235)
+
+## Bugs fixed!
+* Cloud: Stop sending metrics to the cloud output when the cloud returns that you have reached the limit. (#1130)
+* JS: Fail a `check` if an uncaught error is thrown inside of it. (#1137)
+* HTTP: Replace any user credentials in the metric sample tags with `*` when emitting HTTP metrics. (#1132)
+* WS: Many fixes:
+  - return an error instead of panicking if an error occurs during the making of the websocket connection (#1127)
+  - calling the `error` handler on an error when closing the websocket, instead of calling with a null (#1118)
+  - correctly handle server initiated close (#1186)
+* JSON: Better error messages when parsing JSON fails. Now telling you at which line and row the error is instead of just the offset. Thanks, @openmohan! (#905)
+* HTTP: Use Request's `GetBody` in order to be able to get the body multiple times for a single request as needed in 308 redirects of posts and if the server sends GOAWAY with no error. (#1093)
+* JS: Don't export internal go struct fields of script options.(#1151)
+* JS: Ignore `minIterationDuration` for `setup` and `teardown`. (#1175)
+* HTTP: Return error on any request that returns 101 status code as k6 currently doesn't support any protocol upgrade behaviour. (#1172)
+* HTTP: Correctly capture TCP reset by peer and broken pipe errors and give them the appropriate `error_code` metric tag values. (#1164)
+* Config: Don't interpret non-`K6_` prefixed environment variables as k6 configuration, most notably `DURATION` and `ITERATIONS`. (#1215)
+* JS/html: `Selection.map` was not wrapping the nodes it was outputting, which lead to wrongly using the internal `Goquery.Selection` instead of k6's `Selection`. Thanks to @MMartyn for reporting this! (#1198)
+* HTTP: When there are redirects, k6 will now correctly set the cookie for the current URL, instead of for the one the current response is redirecting to. Thanks @dimatock! (#1201)
+* Cloud: Add token to make calls to the cloud API idempotent. (#1208)
+* Cloud: Improve aggregation of HTTP metrics for requests to different URLs, but with the same explicitly set `name` tag. (#1220)
+* Cloud: Fix a bug where you weren't able to run a script, outputting to cloud, if it was using the shortcut urls for github/cdnjs. (#1237)
+
+
+## Internals
+
+* CI: Stop using external service for testing websockets (#1138) and remove the last use of the external `httpbin.org`. (#1213)
+* CI: Move to go 1.13 and back and to 1.13 again after a bug was fixed in the stdlib. (#1145, #1157, #1177)
```

Comment:
hmm I think we should mention that k6 v0.26.0 itself was built with Go 1.13

Reply:
the CI changes are important, but it's more important what we build the actual release with... also, we should probably increment this version: https://github.com/loadimpact/k6/blob/c5f6881d37c44b90b999ac2db4081d5614ab7aaa/README.md#L102